Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN \
 
 RUN \
     apt-get -y update && \
-    apt-get -y install mydumper && \
+    apt-get -y install mydumper git sudo && \
```

Comment:
I introduced the hooks to be able to create a simple script that commits the changes after every backup automatically. So it was required to have `git` available inside the docker.